common/app/agents/DeeplyReadAgent.scala

```diff
@@ -77,6 +77,8 @@ class DeeplyReadAgent(contentApiClient: ContentApiClient, ophanApi: OphanApi) ex
 
     val contentFormat: ContentFormat = ContentFormat(content.design, content.theme, content.display)
 
+    val faciaContentVersion = FaciaContentConvert.contentToFaciaContent(content)
```

Comment:
this version was being generated inline on line 108 but I've pulled it out here because we're reusing it for both the `discussion` and `showQuotedHeadline` values now.